src/test/java/teammates/test/cases/browsertests/FeedbackMcqQuestionUiTest.java

```diff
@@ -59,6 +62,23 @@ public void testNewQuestionFrame() {
         feedbackEditPage.clickNewQuestionButton();
         feedbackEditPage.selectNewQuestionType("MCQ");
         assertTrue(feedbackEditPage.verifyNewMcqQuestionFormIsDisplayed());
+
+        ______TS("MCQ: Check UI after cancelling and add new MCQ question again");
+
+        feedbackEditPage.clickGenerateMcqOptionsCheckbox(NEW_QUESTION_INDEX);
+        assertFalse(feedbackEditPage.isElementVisible("mcqChoiceTable--1"));
```

Comment:
How about `mcqChoiceTable- + NEW_QUESTION_INDEX`? Is it more clear? What do u think?

Reply:
That will require to convert `NEW_QUESTION_INDEX` to string so might get messy. 
Moreover, since its to denote an html tag, I think just writing the whole `id` of element as one string is more clearer as is done in various other places in this file

Reply:
Alright. Let's leave it there. Note that the string is clear to u because u know that `-1` is the new question index and `-` is the delimiter. I don't think this file doesn't a good job on that.